mesonbuild/modules/pkgconfig.py

```diff
@@ -120,8 +120,15 @@ def _process_libs(self, libs, public):
                 processed_reqs.append(obj.generated_pc)
             elif isinstance(obj, dependencies.PkgConfigDependency):
                 if obj.found():
-                    processed_reqs.append(obj.name)
-                    self.add_version_reqs(obj.name, obj.version_reqs)
+                    if obj.static:
+                        # in case of embedded static dependency
+                        for arg in obj.link_args:
+                            if not os.sep in arg:
```

Comment:
__[Flake8]__ test for membership should be 'not in' ([see on Sider](https://sider.review/gh/repos/19784232/pulls/6393#issue-5010321))

<details>

| Rule |
| - |
| E713 |

</details>
<sub>posted by Sider</sub>